### DISCUSSION TITLE
Usage of personal access token for authentification

### DISCUSSION THREAD

**Discussion by christianlupus** [community/community#6240] [Mobile] [answered]
Is it possible to use a PAT for authentication a mobile device?
I consider this favorable over the real password as I can revoke the token if I lost my phone.

I tried and it seems not to be accepted by the mobile site. However, I am not sure, if I did everything correctly and thus wanted to ask.

**Comment by taki-on** [comment, marked as answer]
Hi! We currently do not support an auth path with PAT on Mobile. Thus, you should use your password to login on Mobile. I will pass the idea to the team though :)
